pycbc/workflow/coincidence.py

```diff
@@ -655,3 +690,32 @@ def setup_multiifo_interval_coinc(workflow, hdfbank, trig_files, stat_files,
 
     logging.info('...leaving coincidence ')
     return statmap_files
+
+def select_files_by_ifo_combination(ifocomb, insps):
+    """
+    This function selects the inspiral files for a given ifo combination
+    """
+    # Use only the inspiral files required for the detector combination
+    inspcomb = FileList()
+    for ifo, fi in zip(*insps.categorize_by_attr('ifo')):
+        if ifo in ifocomb:
+            inspcomb += fi
+
+    return inspcomb
+
+def get_ordered_ifo_list(ifocomb, ifo_ids):
+    """
+    This function sorts the combination of ifos (ifocomb) based on the given
+    precedence list (ifo_ids dictionary) and returns the first ifo as pivot
+    the second ifo as fixed, and the ordered list joined as a string.
+    """
+    # combination_prec stores precedence info for the detectors in the combination
+    combination_prec = {}
```

Comment:
Think these 3 lines could be compressed into 1 via `combination_prec = {ifo : ifo_ids[ifo] for ifo in ifocomb}` (dictionary comprehension)